R/shiny.R

```diff
@@ -1269,7 +1269,9 @@ ShinySession <- R6Class(
         ext <- tools::file_ext(filename)
         if (nzchar(ext))
           ext <- paste(".", ext, sep = "")
-        tmpdata <- tempfile(fileext = ext)
+        tmpdata <- tempfile( tmpdir = tempdir(), fileext = ext)
+        snapshot <- utils::fileSnapshot(tempdir())
```

Comment:
(as we discussed over vchat) the test should simply change to `file.exists(tmpdata)`, no need to snapshot and compare.